AzureMonitorAgent/ama_tst/modules/install/check_ama.py

```diff
@@ -1,25 +1,28 @@
 import re
 import requests
-from lxml import html
+import xml.dom.minidom
 
 from error_codes import *
 from errors      import error_info, get_input
-from helpers     import get_package_version, check_internet_connect
+from helpers     import get_package_version
 
 AMA_URL = 'https://docs.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-extension-versions'
 
 def get_curr_ama_version(ama_version):
-    try:
-        r = requests.get(AMA_URL)
-        root = html.fromstring(r.text)
-        versions = root.xpath('//table/tbody/tr/td[4]/text()')
-
-        for version in versions:
-            version = re.sub('[A-Za-z ]+', '', version)
-            if (version == ''):
-                continue
-            if (comp_versions_ge(version, ama_version)):
-                return (version, None)
+    try:           
+        r = requests.get('https://docs.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-extension-versions')
```

Comment:
Small nitpick, but would recommend using `AMA_URL` here since you define it above